github/handlers/ExecuteBlockActionHandler.ts

```diff
@@ -735,11 +735,18 @@ export class ExecuteBlockActionHandler {
                     let value: string = context.getInteractionData().value as string;
                     let splittedValues = value?.split(" ");
                     let { user } = await context.getInteractionData();
+                    let accessToken = await getAccessTokenForUser(this.read, user, this.app.oauth2Config) as IAuthData;
+                    this.app.getLogger().debug(accessToken.token)
```

Comment:
We can remove the log statement that leaks access token. We should be using `accessToken?.token` wherever we think  access token may be a `NULL` value.